From 036cfe4c599b53c438128089f26f5ca54d4350e1 Mon Sep 17 00:00:00 2001
From: Vincent Untz <vuntz@gnome.org>
Date: Mon, 8 Nov 2010 13:42:26 -0500
Subject: [PATCH] Use MRN_SHADOW_* instead of GTK_SHADOW_*

This is needed to avoid warnings when comparing of MurrineShadowType.

https://bugzilla.gnome.org/show_bug.cgi?id=634348
---
 src/murrine_draw.c      |    8 ++++----
 src/murrine_draw_rgba.c |    8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/murrine_draw.c b/src/murrine_draw.c
index 5c211b4..318511b 100644
--- a/src/murrine_draw.c
+++ b/src/murrine_draw.c
@@ -2775,13 +2775,13 @@ murrine_draw_radiobutton (cairo_t *cr,
 	const MurrineRGB *bg = &colors->base[0];
 	gboolean inconsistent = FALSE;
 	gboolean draw_box = !checkbox->in_menu;
-	gboolean draw_bullet = (checkbox->shadow_type == GTK_SHADOW_IN);
+	gboolean draw_bullet = (checkbox->shadow_type == MRN_SHADOW_IN);
 	int roundness = width+height;
 	double highlight_shade_new = widget->highlight_shade;
 	double lightborder_shade_new = widget->lightborder_shade;
 	MurrineGradients mrn_gradient_new = widget->mrn_gradient;
 
-	inconsistent = (checkbox->shadow_type == GTK_SHADOW_ETCHED_IN);
+	inconsistent = (checkbox->shadow_type == MRN_SHADOW_ETCHED_IN);
 	draw_bullet |= inconsistent;
 
 	if (widget->state_type == GTK_STATE_INSENSITIVE)
@@ -2919,13 +2919,13 @@ murrine_draw_checkbox (cairo_t *cr,
 	const MurrineRGB *bg = &colors->base[0];
 	gboolean inconsistent = FALSE;
 	gboolean draw_box = !checkbox->in_menu;
-	gboolean draw_bullet = (checkbox->shadow_type == GTK_SHADOW_IN);
+	gboolean draw_bullet = (checkbox->shadow_type == MRN_SHADOW_IN);
 	int roundness = CLAMP (widget->roundness, 0, 2);
 	double highlight_shade_new = widget->highlight_shade;
 	double lightborder_shade_new = widget->lightborder_shade;
 	MurrineGradients mrn_gradient_new = widget->mrn_gradient;
 
-	inconsistent = (checkbox->shadow_type == GTK_SHADOW_ETCHED_IN);
+	inconsistent = (checkbox->shadow_type == MRN_SHADOW_ETCHED_IN);
 	draw_bullet |= inconsistent;
 
 	if (widget->state_type == GTK_STATE_INSENSITIVE)
diff --git a/src/murrine_draw_rgba.c b/src/murrine_draw_rgba.c
index 5743b2c..05cb5ea 100644
--- a/src/murrine_draw_rgba.c
+++ b/src/murrine_draw_rgba.c
@@ -1821,13 +1821,13 @@ murrine_rgba_draw_radiobutton (cairo_t *cr,
 	const MurrineRGB *bg = &colors->base[0];
 	gboolean inconsistent = FALSE;
 	gboolean draw_box = !checkbox->in_menu;
-	gboolean draw_bullet = (checkbox->shadow_type == GTK_SHADOW_IN);
+	gboolean draw_bullet = (checkbox->shadow_type == MRN_SHADOW_IN);
 	int roundness = width+height;
 	double highlight_shade_new = widget->highlight_shade;
 	double lightborder_shade_new = widget->lightborder_shade;
 	MurrineGradients mrn_gradient_new = widget->mrn_gradient;
 
-	inconsistent = (checkbox->shadow_type == GTK_SHADOW_ETCHED_IN);
+	inconsistent = (checkbox->shadow_type == MRN_SHADOW_ETCHED_IN);
 	draw_bullet |= inconsistent;
 
 	if (widget->state_type == GTK_STATE_INSENSITIVE)
@@ -1967,13 +1967,13 @@ murrine_rgba_draw_checkbox (cairo_t *cr,
 	const MurrineRGB *bg = &colors->base[0];
 	gboolean inconsistent = FALSE;
 	gboolean draw_box = !checkbox->in_menu;
-	gboolean draw_bullet = (checkbox->shadow_type == GTK_SHADOW_IN);
+	gboolean draw_bullet = (checkbox->shadow_type == MRN_SHADOW_IN);
 	int roundness = CLAMP (widget->roundness, 0, 2);
 	double highlight_shade_new = widget->highlight_shade;
 	double lightborder_shade_new = widget->lightborder_shade;
 	MurrineGradients mrn_gradient_new = widget->mrn_gradient;
 
-	inconsistent = (checkbox->shadow_type == GTK_SHADOW_ETCHED_IN);
+	inconsistent = (checkbox->shadow_type == MRN_SHADOW_ETCHED_IN);
 	draw_bullet |= inconsistent;
 
 	if (widget->state_type == GTK_STATE_INSENSITIVE)
-- 
1.7.3.1